platform-sdk/swirlds-state-impl/src/main/java/com/swirlds/state/merkle/StateUtils.java

```diff
@@ -503,4 +504,14 @@ public static byte[] createVirtualMapKeyBytesForKV(
 
         return byteBuffer.array();
     }
+
+    public static <V> VirtualMapValue getVirtualMapValue(
+            @NonNull final String serviceName, @NonNull final String stateKey, final V value) {
+        return new VirtualMapValue(new OneOf<>(
+                VirtualMapValue.ValueOneOfType.fromProtobufOrdinal(getValidatedStateId(serviceName, stateKey)), value));
+    }
+
+    public static <V> VirtualMapValue getVirtualMapValueQueueState(final V value) {
```

Comment:
`getQueueStateVirtualMapValue()` sounds better than `getVirtualMapValueQueueState()` to me. The arg can be changed to `QueueState` rather than (generic) `V`

Reply:
Done